src/containers/Folder/components/FoldersContainer/FoldersContainer.tsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from 'react';
+import FoldersView from '../FoldersView';
+import InfoContainer from '../InfoContainer';
+import { Folder } from '@/types/Folder.types';
+import { DEFAULT_USER_ID } from '@/services/config/default';
+
+const DEFAULT_FOLDER = {
+  id: 0,
+  name: '전체',
+};
+
+interface Props {
+  folders: Folder[];
+  initialFolderId: number;
+  setFolderLinks: (nextFolderId: number) => void;
+}
+
+function FoldersContainer({ folders, initialFolderId, setFolderLinks }: Props) {
+  const initialSelectedFolder = folders?.find(
+    (folder) => folder.id === initialFolderId
+  );
+
+  const [selectedFolder, setSelectedFolder] = useState<Folder>(DEFAULT_FOLDER);
+
+  const onFolderButtonClick = (folders: Folder) => {
+    setFolderLinks(folders.id);
+    setSelectedFolder(folders);
+  };
```

Comment:
폴더 하나에 대한 내용이기 때문에 단수형을 써주시면 더 좋을 것 같네요.
```suggestion
  const onFolderButtonClick = (folder: Folder) => {
    setFolderLinks(folder.id);
    setSelectedFolder(folder);
  };
```